sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1168,7 +1167,7 @@
     "/messages/serviceBound/feedback/{lockToken}": {
       "delete": {
         "summary": "This method completes a feedback message.",
-        "description": "This method completes a feedback message. The lockToken obtained when the message was received must be provided to resolve race conditions when completing, a feedback message. A completed message is deleted from the feedback queue. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "This method completes a feedback message. A completed message is deleted from the feedback queue. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging for more information.",
```

Comment:
We have been getting rid of the `summary` in the swagger, since autorest doesn't care about it, and it isn't getting referenced anywhere in the auto-gen code.

Reply:
I agree and I believe we missed to discuss this in detail on my PR. We need to ask the Service team how it gets generated. Otherwise it'll come back. These are just suggestions to the team and we're not making any hand edits yet.

Reply:
That's true, I missed it for this one.